webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -113,7 +113,6 @@ def test_show_bug_link_with_cache(self):
 
 
 class Test_generate_create_bug_url:
-    EMPTY_PARSED_DUMP = {}
```

Comment:
Most of the changes in this file are because we changed the signature for `generate_create_bug_url` removing the raw crash and request and using the `json_dump` value from the processed crash.